src/Commands/HoneybadgerCheckinsSyncCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Honeybadger\HoneybadgerLaravel\Commands;
+
+use Exception;
+use Honeybadger\Contracts\CheckinsSync;
+use Honeybadger\Contracts\Reporter;
+use Illuminate\Console\Command;
+
+class HoneybadgerCheckinsSyncCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'honeybadger:checkins-sync';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Synchronize checkins to Honeybadger';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle(CheckinsSync $checkinsManager)
+    {
+        try {
+            $localCheckins = config('honeybadger.checkins', []);
+            $result = $checkinsManager->sync($localCheckins);
+            $this->info('Checkins were synchronized with Honeybadger.');
+            $this->table(['Id', 'Name', 'Schedule Type', 'Cron Schedule', 'Cron Timezone', 'Grace Period', 'Status'], array_map(function ($checkin) {
+                return [
+                    $checkin->id,
+                    $checkin->name,
+                    $checkin->scheduleType,
+                    $checkin->cronSchedule,
+                    $checkin->cronTimezone,
+                    $checkin->gracePeriod,
+                    $checkin->isDeleted() ? '❌ Removed' : '✅ Synchronized',
+                ];
+            }, $result));
+        } catch (Exception $e) {
+            $this->error($e->getMessage());
```

Comment:
Tbh, I don't think this is needed. My recommendation: if there are specific errors that we can anticipate (if any), then handle only those—print a useful message + help, and let all others crash.

The reason is that by printing only the exception message here, you're effectively hiding information that might be helpful to the user (the stack trace). That's worse than just letting it crash (in which case it will print a stack trace).

Reply:
> Tbh, I don't think this is needed. My recommendation: if there are specific errors that we can anticipate (if any), then handle only those—print a useful message + help, and let all others crash.

I wasn't sure how to tackle this and I went for the approach that was already implemented for the rest of the package: let the user control whether an exception in the honeybadger will cause the script to crash or simply log a message.
If there is a `ServiceException` is thrown, `$e->getMessage()` provides a helpful message to the user.

> The reason is that by printing only the exception message here, you're effectively hiding information that might be helpful to the user (the stack trace). That's worse than just letting it crash (in which case it will print a stack trace).

Good point, will correct!

In conclusion, here's what I will do:
- I will catch `ServiceException` and log a warning message
- I will let other exceptions crash the process and (potentially) blocking the deployment script.

Reply:
> I will let other exceptions crash the process and (potentially) blocking the deployment script.

Actually, I'm not sure about this. If there are other kind of exceptions thrown, it probably means that they are out of our control (and quite possible the user's control). In that case, why block the deployment script? What do you think?

Reply:
> I will catch ServiceException and log a warning message

This is a good approach. But that's not a warning. If my quota is up and my checkins aren't syncing, that's bad.

My take is that you should catch expected exceptions (`ServiceException`), log an error and fail (probably something like `$this->fail()`). Other errors should also fall automatically (but we should have good testing to limit unexpected crashes).

A user can always bypass this by adding `|| true` in their script (command returns a nonzero exit code, `|| true` makes it always return true).

Reply:
There's no `$this->fail()` for commands, so I just let them all fail, which prints a helpful error message (when it's a `ServiceException`).